ui/component/walletExport/view.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import Button from 'component/button';
+import { Form, FormField } from 'component/common/form';
+import Card from 'component/common/card';
+import Lbry from 'lbry';
+import { clipboard } from 'electron';
+import * as ICONS from 'constants/icons';
+import FileExporter from 'component/common/file-exporter';
+
+type Props = {
+  toast: (string, boolean) => void,
+};
+
+function WalletExport(props: Props) {
+  const { toast } = props;
+
+  const [password, setPassword] = React.useState();
+  const [data, setData] = React.useState();
+  const [fetching, setFetching] = React.useState();
+  const getData = async (password) => {
+    setFetching(true);
+    const data = await Lbry.sync_apply({ password });
+    setFetching(false);
+    setData(data);
+  };
```

Comment:
- [ ] The modal will be stuck grayed out when `sync_apply` fails (e.g. network issues).  Another way to replicate is to not provide a password:

<img width="950" alt="image" src="https://user-images.githubusercontent.com/64950861/179445706-f7bb8a88-6278-4c08-bc33-731e735b3b91.png">